docs/product/security/ip-ranges.mdx

```diff
@@ -50,13 +50,21 @@ In some circumstances the Hosted Sentry infrastructure might send HTTP requests
 
 Sentry uses the following IP addresses to make outbound requests:
 
+US Region
```

Comment:
```suggestion
US Data Storage Location
```
Feel free to disagree if region feels better here, I don't feel strongly, but we're referring to them as data storage locations in the rest of our user-facing docs